.github/kokoro/steps/vtr-test.sh

```diff
@@ -68,8 +68,11 @@ find vtr_flow/benchmarks/titan_blif/ -type f -not -name 'README.*' -delete
 # Removing ISPD, Titan and Symbiflow tarballs
 find . -type f -regex ".*\.tar\.\(gz\|xz\)" -delete
 
+#Gzip output files from vtr_reg_nightly tests to lower working directory disk space
+find vtr_flow/tasks/regression_tests/vtr_reg_nightly/ -type f -print0 | xargs -0 -P $(nproc) gzip -v
+
 # Make sure working directory doesn't exceed disk space limit!
-echo "Working directory size: $(du -sh)"
+echo "Working directory size: $(du -h)"
```

Comment:
This should still be `du -sh` because the goal is to get a print like:

```
Working directory size: 5 GB
```

If you want to get a detailed disk space report, it should probably be another line.  If you want a detailed report, I recommend limiting the depth to avoid a very verbose output.

Reply:
Ok, will make that change